### ISSUE TITLE
Activities' redesign [enhancement]

### ISSUE THREAD

**Issue by TheSpartan94** [nvllsvm/GZDoom-Android#32]
Layouts for the main wad selection menu & settings seem really uninspired and raw. The activities should be a little redeisgned to enhance usability & _look and fell_ of the application.

I can help the developer in doing so, if he needs.

**Comment by nvllsvm** [comment]
Agreed. I'm toying with a complete rewrite of the UI, but have nothing to share yet.
